package/src/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts

```diff
@@ -56,17 +77,16 @@ export const useChannelPreviewDisplayName = <
 ) => {
   const { client } = useChatContext<StreamChatGenerics>();
 
-  const currentUserId = client.userID;
-  const members = channel?.state.members;
+  const currentUserId = client?.userID;
+  const members = channel?.state?.members;
   const numOfMembers = Object.keys(members || {}).length;
   const channelName = channel?.data?.name;
-  const maxCharacterLength = characterLength || maxCharacterLengthDefault;
-
+  const maxCharacterLength = characterLength || DEFAULT_MAX_CHARACTER_LENGTH;
   const [displayName, setDisplayName] = useState(
     getChannelPreviewDisplayName({
       channelName,
+      characterLimit: maxCharacterLength,
```

Comment:
This looks like the result of a mass refactor done by an IDE - would it be better to rename the local `maxCharacterLength` to characterLimit? We should also use consistent naming so we have either `characterLimit` or `characterLength` naming and not both, that can get confusing. I think `characterLength` isn't that great a name though, it's not the length of the characters it holds.